README.md

```diff
@@ -1,5 +1,6 @@
 # MusicBrainz UserScripts
 
+* [Import Facebook events to MusicBrainz](#facebook_events_importer)
```

Comment:
It looks like the list is alphabetically sorted, so this entry should be at the top of list. ;)